services/movie.py

```diff
@@ -6,9 +7,13 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: str = None,
```

Comment:
If argument can be `None`, you should show it in type hint
```suggestion
    title: str | None = None,
```

Reply:
Fixed. You always write about it and I always forget. Not good, not good at all, as Jack Reacher says))